BuilderImage.Dockerfile

```diff
@@ -23,32 +23,33 @@ RUN corepack enable
 
 # Install Browsers.
 #Chrome
-ARG CHROME_VERSION="120.0.6099.129"
-RUN wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_$CHROME_VERSION-1_amd64.deb && \
-    dpkg -i google-chrome-stable_$CHROME_VERSION-1_amd64.deb || apt-get -f install -y
+RUN wget -qO- https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -O google-chrome-stable_current_amd64.deb && \
+    dpkg -i google-chrome-stable_current_amd64.deb || apt-get -f install -y
 
-
-#Edge
+# Install Browsers.
+# Edge
 RUN curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg && \
     install -o root -g root -m 644 microsoft.gpg /etc/apt/trusted.gpg.d/ && \
-    sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/edge stable main" > /etc/apt/sources.list.d/microsoft-edge-dev.list' && \
+    sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/edge stable main" > /etc/apt/sources.list.d/microsoft-edge.list' && \
     rm microsoft.gpg && \
     apt-get update -y && \
     apt-get install -y microsoft-edge-stable
 
-
-#Firefox
-ARG FIREFOX_VERSION="121.0"
-RUN apt-get -qqy --no-install-recommends install firefox \
+# Install Browsers.
+# Firefox
+RUN apt-get update && apt-get install -qqy --no-install-recommends wget gnupg ca-certificates \
```

Comment:
do we need `apt-get update`?

Reply:
Probably not. It's redundant as we're calling it earlier in the process as well.